File: UnacademyFreeLiveClass/Pattern2.cpp
#include <iostream>
using namespace std;

int main() {

	int n;
	cin>>n;

	int row = 1;

	while(row <= n) {
		// this will help us to iterate on each row

		int spaces = 1;
		while(spaces <= n - row) {
			cout<<" ";
			spaces+=1;
		}

		int stars = 1;
		while(stars <= 2*row - 1) {
			cout<<"*";
			stars += 1;
		}

		cout<<"\n";
		row += 1;
	}

	return 0;
}

File: UnacademyCourse/DPInterviewPrepMay2021/Lecture7/pattern2.js
let n = 5;

for(let row = 1; row <= n; row++) {
    let str = "";
    for(let spaces = 1; spaces <= n-row; spaces++) {
        str += " "
    }
    for(let stars = 1; stars <= 2*row-1; stars++) {
        str += "*";
    }
    console.log(str);
}